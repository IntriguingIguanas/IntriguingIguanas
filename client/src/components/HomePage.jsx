import React from 'react'
import Lobby from './Lobby.jsx'
import BoardPage from './BoardPage.jsx'
import { Link } from 'react-router'

export class HomePage extends React.Component {
  constructor(props) {
    super(props)
  }

  render() {
    return (
    <div>
      {/* ----- NAV BAR ----- */}
      <div className="ui blue inverted stackable menu">
        <div className="ui container">
          <img className='logo' src="./Logo.png"></img>
        </div>
        <a className="item">
          <Link to='/login'><i className="user icon"></i>LOG IN</Link>
        </a>
        <a className="item">
          <Link to='/signup'><i className="add user icon"></i>SIGN UP</Link>
        </a>
      </div>

      <div className="jumbotron">
        <h1 className="homepage">Root.io is your one stop shop with all the tools necessary for live and remote collaboration.</h1>
        <h2 className="homepage">With shared project boards and video chat, your team can manage tasks seamlessly.</h2>
        <button className="ui primary button sign-up" onClick={ this.props.route.sendToSignup }><i className="add user icon"></i>SIGN UP</button>
        <h4 onClick={ this.props.route.sendToLogin }>Already a Root.io user? Log in.</h4>
      </div>
    </div>
    )
  }
}

export default HomePage

