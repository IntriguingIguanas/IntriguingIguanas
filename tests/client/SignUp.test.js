import React from 'react'
import ReactDOM from 'react-dom'
import { App } from 'App'
import { shallow, mount } from 'enzyme'
import HomePage from 'HomePage'


describe('SignUpPage', () => {
  it('should be able to run tests', () => {
    expect(1 + 2).toEqual(3);
  });
});
