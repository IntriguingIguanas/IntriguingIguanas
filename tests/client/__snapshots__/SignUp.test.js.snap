// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignUpPage should render signup input forms 1`] = `
<div>
  <form
    onSubmit={[Function]}
  >
    <label>
      Username:
      <input
        onChange={[Function]}
        type="text"
        value=""
      />
    </label>
    <label>
      Email:
      <input
        onChange={[Function]}
        type="text"
        value=""
      />
    </label>
    <label>
      Password:
      <input
        onChange={[Function]}
        type="text"
        value=""
      />
    </label>
    <input
      type="submit"
      value="Signup"
    />
  </form>
</div>
`;
